src/pages/Facility/services/inventory/externalSupply/deliveryOrder/DeliveryOrderShow.tsx

```
@@ -620,6 +621,21 @@ export function DeliveryOrderShow({
                   </div>
                 </div>
               </div>
+              {deliveryOrder.created_by && (
+                <div>
+                  <label className="text-sm font-medium text-gray-700">
+                    {t("created_by")}
+                  </label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)